dropwizard-jackson/src/test/java/io/dropwizard/jackson/JacksonTest.java

```diff
@@ -48,11 +48,11 @@ void objectMapperSerializesNullValues() throws IOException {
     }
 
     @Test
-    void objectMapperIgnoresUnknownProperties() {
-        assertThatCode(() ->
-            Jackson.newObjectMapper()
+    void objectMapperIgnoresUnknownProperties() throws JsonProcessingException {
+        assertThat(Jackson.newObjectMapper()
                 .readValue("{\"unknown\": 4711, \"path\": \"/var/log/app/server.log\"}", LogMetadata.class)
-        ).doesNotThrowAnyException();
+                .path)
+            .hasToString("/var/log/app/server.log");
```

Comment:
@rhowe This change breaks the test on Windows: https://ci.appveyor.com/project/dropwizard/dropwizard/builds/40883668#L6261

Reply:
Honestly, these toy OSes. When will we learn? I'll take a look, thanks